hackathon-logistics/README.md

```diff
@@ -1,4 +1,30 @@
-# What to expect at a Hackathon
+# Hackathon? What does that mean?
 
+There are so many guides out there about what hackathons are like and what to expect (some of them even by our own Freetail alumni!). Here are some of my favorites:
+- https://medium.com/freetail-hackers/hacking-your-first-hackathon-e2f07060de5d
+- https://blog.usejournal.com/a-hackers-guide-to-hackathons-e4b0e5f4c757
+- https://medium.com/tfogo/hackathons-are-for-beginners-77a9c9c0e000
 
+But it's difficult to actually put in words what a Hackathon experience is like, regardless, here's my best attempt.
 
+#### So what is a Hackathon?
+A hackathon is an invention marathon, where people come together, using their technical and creative skills to create solutions for everyday problems. It's a space where people jump off each others energy and create something out of nothing but a spark of an idea. What I like the most about hackathons is just the energy of innovation, knowing that everyone there is  making something they are passionate about and are excited about.
+      
+#### What should I do before I come to the event?
+There's very little you have to do before the actual event--most of the magic happens there. Just be sure to look over the hackathon's pre-event email and follow whatever instructions they have for you. I would also make sure you are well rested the night before and ready to create the next day. If you would like, I would recommend finding a team before the event starts, whether that's friends that you registered with or people that you have connected with online. It's also good to brainstorm some ideas (more on this linked [here](https://github.com/kdesai2018/ultimate-hackathon-starting-guide/blob/master/ideation/README.md)) and come in with a project idea. 
+
+#### What should I expect?
+Most hackathons are held in a big room of some sort with tables for teams to be at. There will typically be meals provided, lots of snacks at all times, and events throughout. At HackTX we had events like a CTF, workshops, and a paper airplane making contest. Definitely check these out--they're lots of fun and you can meet some really cool people. Talk to your peers and mentors--we are here to help! There are so many resources availible to you at a hackathon: companies to talk to (and swag to get;), mentors, organizors, and more. Most of the time people will be working and some people will event stay up through the entire night. Although this can be part of the hackathon experience, don't feel like you have to do this. Sleep is important to productivity too. At the end, most hackathons will have teams demo to judges and give the top teams prizes. I would recommend demoing regardless of how your project turns out. It's a good experience just to talk through your idea, feel proud of what you made, and explain the process to a judge. Most of all, you should expect to have fun! Expect to learn, meet new friends, and enjoy a great environment and driven community. 
```

Comment:
```suggestion
Most hackathons are held in a big room of some sort with tables for teams to be at. There will typically be meals provided, lots of snacks at all times, and events throughout. At HackTX we had events like a CTF ([Capture The Flag](https://ctfd.io/whats-a-ctf/)), workshops, and a paper airplane making contest. Definitely check these out--they're lots of fun and you can meet some really cool people. Talk to your peers and mentors--we are here to help! There are so many resources available to you at a hackathon: companies to talk to (and swag to get), mentors, organizers, and more. Most of the time people will be working and some people will even stay up the entire night. Although this can be part of the hackathon experience, don't feel like you have to do this. **Sleep is important for productivity too**. At the end, most hackathons will have teams demo to judges and give the top teams prizes. I would recommend demoing regardless of how your project turns out. It's a good experience just to talk through your idea, feel proud of what you made, and explain the process to a judge. Most of all, you should expect to have fun! Expect to learn, meet new friends, and enjoy a great environment and driven community. 
```